rskj-core/src/main/java/org/ethereum/rpc/FilterManager.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2018 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.ethereum.rpc;
+
+import org.ethereum.core.Block;
+import org.ethereum.core.Transaction;
+
+import java.util.ArrayList;
+import java.util.Hashtable;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Created by ajlopez on 17/01/2018.
+ */
+public class FilterManager {
+    private static final long filterTimeout = 5 * 60 * 1000; // 5 minutes in milliseconds
+    private static final long filterCleanupPeriod = 1 * 60 * 1000; // 1 minute in milliseconds

```

Comment:
The same about the usage of long, whenever we can we should change it for Duration